kpi-tracker/src/main.rs

```diff
@@ -1,5 +1,4 @@
 use core::fmt;
-use std::collections::{BTreeMap, BTreeSet, HashMap, HashSet};
 
```

Comment:
This is a nitpick, but can we not have empty lines between imports. And let the formatter sort them.